lib/experiments-page.php

```diff
@@ -163,6 +163,18 @@ function gutenberg_initialize_experiments_settings() {
 		)
 	);
 
+	add_settings_field(
+		'gutenberg-block-comment',
+		__( 'Block Comments', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Enable multi-user commenting on blocks in post editor', 'gutenberg' ),
```

Comment:
We’ve made this update based on the feedback provided here: https://github.com/WordPress/gutenberg/pull/60622#issuecomment-2309620757.

Reply:
I don't think we should be adding this setting here, and hardcode the comment type. We should generalise it to work either with all comment types, or add an (experimental) option/setting to the comment type registration and use that setting (instead of hardcoding anything).